charts/dynamoai-api/values.yaml

```diff
@@ -94,6 +94,32 @@ api:
   # ref: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.22/#envvar-v1-core
   extraEnv: []
 
+  # TODO: move these to common configs.
+  authProvider: keycloak  # or cognito
+  keycloakEnv:
+    - name: KEYCLOAK_REALM
+      value: dynamo-ai
+    - name: KEYCLOAK_API_CLIENT_ID
+      value: api
+    - name: KEYCLOAK_BASE_URL
+      value: ''
+    - name: KEYCLOAK_API_CLIENT_SECRET
+      value: ''
+    - name: 'API_AUTH_MODE'
+      value: 'keycloak-and-key'
+
+  cognitoEnv:
+    - name: 'API_AUTH_MODE'
+      value: 'cognito-and-key'
+
+  storageProvider: minio # or s3
+  minioEnv:
+    - name: S3_ENDPOINT
```

Comment:
Can we add another variable here `AWS_ENDPOINT_URL_S3`?
It's value will be same as S3_ENDPOINT and we might deprecate S3_ENDPOINT (I'm trying that out)